m2cgen/interpreters/elixir/code_generator.py

```diff
@@ -43,6 +49,7 @@ def add_function_def(self, name, args, is_scalar_output):
         self.add_code_line(function_def)
 
         self.increase_indent()
+        self.add_code_line("input = list_to_binary(input)")
```

Comment:
Is there a less hacky way to do this?

The name of the overridden method `add_function_def` suggests this is a generic function definition, not the "score" function.

Reply:
Sorry for the late response - was busy these days. TBH, I don't want to delay this PR and bother you with any other investigations or significant code refactoring. I think this approach is OK for now. Later I can refactor this.

Only one thing bothers me, 
> But there's a flaky test and I'm not sure what to do (failed 1/5 times):

do you mean that right now that test can fail any time? It's quite odd because I believe all random seeds are fixed in tests... BTW, I remember we had another problem with this particular test: #205. If so, let's just skip that test for Elixir for now similarly to how tests with missing values (NaNs) are being skipped ring now. WDYT?

Reply:
> do you mean that right now that test can fail any time? It's quite odd because I believe all random seeds are fixed in tests...

yeah, it happened a few times randomly - my perception is that it happens 20% of the time

> If so, let's just skip that test for Elixir for now similarly to how tests with missing values (NaNs) are being skipped ring now. WDYT? 

I tried to do this, but I couldn't figure out how to skip a specific test in the test suite. Would you mind pointing me how to do it?

Reply:
I think we can simply deselect that test for now here
https://github.com/BayesWitnesses/m2cgen/blob/333d808bf3a0cef856ce26507b8f36c12e6ee14d/.ci/test.sh#L17
via `-k` option:

```
pytest -v "-m=$LANG" "-k=not(xgboost_XGBClassifier and elixir and train_model_classification_binary2)" tests/e2e/
```